go/tasks/config/config.go

```diff
@@ -18,12 +16,7 @@ var (
 
 // Top level plugins config.
 type Config struct {
-	EnabledPlugins []string `json:"enabled-plugins" pflag:"[]string{\"*\"},List of enabled plugins, default value is to enable all plugins."`
-}
-
-func (cfg Config) IsEnabled(pluginToCheck string) bool {
-	return cfg.EnabledPlugins != nil && len(cfg.EnabledPlugins) >= 1 &&
-		(cfg.EnabledPlugins[0] == "*" || utils.Contains(cfg.EnabledPlugins, pluginToCheck))
+	EnabledPlugins []string `json:"enabled-plugins" pflag:"[]string{\"*\"},Deprecated - List of enabled plugins, default value is to enable all plugins."`
```

Comment:
```suggestion
	EnabledPlugins []string `json:"enabled-plugins" pflag:",Deprecated - List of enabled plugins, default value is to enable all plugins."`
```

Reply:
The style of setting default values inline is deprecated (haven't been fully removed from all the codebase yet admittedly).

Instead:
```golang
//go:generate pflags Config --default-var=defaultConfig

var (
  defaultConfig = &Config{
    EnabledPlugins: []string{},
  }

  rootSection = config.MustRegisterSection(configSectionKey, &defaultConfig)
)